packages/frontend-shared/src/gql-components/HeaderBarContent.cy.tsx

```diff
@@ -24,15 +26,23 @@ describe('<HeaderBarContent />', { viewportWidth: 1000, viewportHeight: 750 }, (
       .should('not.exist')
     })
 
+    cy.get('[data-cy="top-nav-browser-list-item"]')
+    .last().should('not.be.visible')
+    .scrollIntoView().should('be.visible')
```

Comment:
hmm, in light of https://github.com/cypress-io/cypress/issues/877 I'm actually a bit surprised this works. I ran into this exact case in a different context, and asserting visibility did not work.

Reply:
Good catch! I adjusted the test to just check for overflow.

The reason it was passing was because the browsers list (without overflow) is long enough that the last element renders off the screen. If the list was shortened, the assertion would fail.